### PR TITLE
Add note about Wiki Style Renderer in JIRA.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/6910/add-a-note-to-the-jira-integration-that-the-wiki-style-renderer-needs-to-be-enabled-to-properly-view-bb-feedback-reports

Includes a JIRA troubleshooting page that demonstrates how to
appropriately set the renderer for JIRA issue descriptions.

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://124-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://124-bb-docs.surge.sh

File: troubleshooting/jira.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+. Click the **Configure** link to the right of the **Default Field
+  Configuration** label. The **View Field Configuration** screen is
+  displayed:
++
+image:img/screen-jira-administration-field_configuration.png["The View
+Field Configuration screen in JIRA", 1280, 624, role="frame"]
+
+. Scroll down to the **Description** area:
++
+image:img/panel-jira-field_configuration-description.png["The
```
Comment (oldfatcrab) [inline troubleshooting/jira.adoc]: When zoom in this image, the horizontal and vertical ratio is not the same as original... could you take a look at this?

screen shot 2017-11-24 at 10 24 17 am

**Comment by bbdocs** [comment]
See the website here: http://124-bb-docs.surge.sh

**Pull request merged**